lib/mempool/mempool.js

```diff
@@ -284,6 +285,12 @@ class Mempool extends EventEmitter {
         this.untrackClaim(entry);
     }
 
+    // At the chain.tip.height of claimPeriod - 1, we no longer
+    // accept new claims as they will be invalid in the next block.
+    // We also need to clean them up from the mempool.
+    if (nextHeight === this.network.names.claimPeriod)
```

Comment:
tiny nit, the next code block (goosigStop) directly uses `block.height + 1`, that can also use nextHeight right?